src/main.js

```diff
@@ -1,34 +1,50 @@
-var buttonSection = document.querySelector('.button-section');
+var buttonSection = document.querySelector('.button-select-category');
+
+var currentActivity;
+var currentCategory
 
 buttonSection.addEventListener('click', clickHandler);
 
 function clickHandler(event) {
   if (event.target.classList.contains('study')) {
-    changeStudyButton();
+    changeStudyButton(event);
   } else if (event.target.classList.contains('meditate')) {
     changeMeditateButton();
   } else if (event.target.classList.contains('exercise')) {
     changeExerciseButton();
+  } else if (event.target.classList.contains('start-activity')) {
+    newActivity(event);
   }
 }
 
 function changeStudyButton() {
   var studyButton = document.querySelector('.study');
   var studyImage = document.querySelector('#study-icon');
   studyButton.classList.toggle('study-active');
-  studyImage.src = studyImage.src.match("assets/study.svg") ? "assets/study-active.svg" : "assets/study.svg";
+  studyImage.src = studyImage.src.match('assets/study.svg') ? 'assets/study-active.svg' : 'assets/study.svg';
+  currentCategory = 'study';
 }
 
 function changeMeditateButton() {
   var meditateButton = document.querySelector('.meditate');
   var meditateImage = document.querySelector('#meditate-icon');
   meditateButton.classList.toggle('meditate-active');
-  meditateImage.src = meditateImage.src.match("assets/meditate.svg") ? "assets/meditate-active.svg" : "assets/meditate.svg";
+  meditateImage.src = meditateImage.src.match('assets/meditate.svg') ? 'assets/meditate-active.svg' : 'assets/meditate.svg';
+  currentCategory = 'meditate';
 }
 
 function changeExerciseButton() {
   var exerciseButton = document.querySelector('.exercise');
   var exerciseImage = document.querySelector('#exercise-icon');
   exerciseButton.classList.toggle('exercise-active');
-  exerciseImage.src = exerciseImage.src.match("assets/exercise.svg") ? "assets/exercise-active.svg" : "assets/exercise.svg";
+  exerciseImage.src = exerciseImage.src.match('assets/exercise.svg') ? 'assets/exercise-active.svg' : 'assets/exercise.svg';
+  currentCategory = 'exercise';
```

Comment:
I am glad it was easy to make this global variable of currentCategory to obtain which category it was when a use clicks the button.